package/launcher/src/main.rs

```diff
@@ -50,7 +50,16 @@ fn main() {
     // compute deno and deno dom locations (allow them to be defined externally)
     let mut deno_file = path_from_env("QUARTO_DENO");
     if deno_file.as_os_str().is_empty() {
```

Comment:
It looks like the shell exports `QUARTO_DENO` (which then allows `quarto run` to find deno) however the Rust front end doesn't seem to export it (it can do this w/ the other env vars on line 70 below)

Reply:
Ok, I'll change.